tools/build_node.js

```diff
@@ -195,6 +196,8 @@ async function buildNode(options) {
   await buildCJSIndex("index", languages);
   await buildCJSIndex("common", common);
   await buildLanguages(languages, options);
+  await fs.writeFile(`${process.env.BUILD_DIR}/lib/languages/index.js`, generateCJSLanguageIndex());
+  await fs.writeFile(`${process.env.BUILD_DIR}/es/languages/index.js`, generateESLanguageIndex());
```

Comment:
```suggestion
  await fs.writeFile(`${process.env.BUILD_DIR}/lib/languages/index.js`, generateCJSLanguageIndex(languages));
  await fs.writeFile(`${process.env.BUILD_DIR}/es/languages/index.js`, generateESLanguageIndex(languages));
```

The build system languages are a build time choice, so index generation would need to take the build time language list as input.